builtin/logical/pki/fields.go

```diff
@@ -509,6 +517,28 @@ Defaults to 8760 hours (1 year).`,
 		Default: int(defaultTidyConfig.IssuerSafetyBuffer / time.Second), // TypeDurationSecond currently requires defaults to be int
 	}
 
+	fields["acme_account_revocation_safety_buffer"] = &framework.FieldSchema{
```

Comment:
I think this needs to be `revoke_acme_accounts_safety_buffer` based on the below :-)